Content.Client/GameObjects/Components/Mobs/LooksComponent.cs

```diff
@@ -0,0 +1,61 @@
+using Content.Shared.GameObjects.Components.Mobs;
+using Content.Shared.Preferences;
+using Content.Shared.Preferences.Appearance;
+using Robust.Client.GameObjects;
+using Robust.Shared.GameObjects;
+
+namespace Content.Client.GameObjects.Components.Mobs
+{
+    [RegisterComponent]
+    public sealed class LooksComponent : SharedLooksComponent
```

Comment:
I hate this name, but Appearance exists and it's a totally different thing. `HumanoidLooksComponent`? `HumanoidAppearanceComponent`?

Reply:
I'd definitely go with `HumanoidAppearanceComponent`